Parsers/Generate dev shortlink.js

```diff
@@ -0,0 +1,17 @@
+/*
+activation_example:!devlink utah
+regex:(!devlink)|(!shortlink) 
+flags:gmi
+*/
+
+(function(){
+    
+  var term = current.text.replace('!devlink', '').trim();
```

Comment:
You need to do the same for `!shortlink`

You could probably `.replace(/(!devlink)|(!shortlink)/,'').trim()`

Reply:
Updated it.